Data/src/main/java/org/tribuo/data/columnar/extractors/OffsetDateTimeExtractor.java

```diff
@@ -65,8 +88,14 @@ public OffsetDateTimeExtractor(String fieldName, String metadataName, String dat
     @Override
     public void postConfig() {
         super.postConfig();
+        Locale locale;
+        if ((localeLanguage == null) && (localeCountry == null)) {
+            locale = Locale.getDefault(Locale.Category.FORMAT);
+        } else {
+            locale = new Locale(localeLanguage,localeCountry);
```

Comment:
Won't we end up here if one of `localeLanguage` and `localeCountry` are null?
Maybe if either one is null we use the default and log a warning.

Reply:
I changed it so they throw `PropertyException`. The lack of default in locale is to ensure the default behaviour is the same in Tribuo 4.2+ as it was in 4.1 and 4.0.